tests/runtests.pl

```diff
@@ -3200,6 +3201,7 @@ sub subVariables {
 
   $$thing =~ s/%CURL/$CURL/g;
   $$thing =~ s/%PWD/$pwd/g;
+  $$thing =~ s/%POSIX_PWD/$posix_pwd/g;
```

Comment:
But is there a reason to keep the old one? I mean couldn't you just use this method for `%PWD` ?

Reply:
aah, ok. Thanks!